pixo-platform/servicetest/steps.go

```diff
@@ -99,6 +100,64 @@ func (s *ServerTestFeature) InitializeScenario(ctx *godog.ScenarioContext) {
 	ctx.Step(`^the message should contain a "([^"]*)"$`, s.TheMessageShouldContainA)
 	ctx.Step(`^the message should not contain a "([^"]*)"$`, s.TheMessageShouldNotContainA)
 	ctx.Step(`^the message should not be empty$`, s.CheckMessageNotEmpty)
+	ctx.Step(`^the json query "([^"]*)" should not exists in the response$`, s.theJsonQueryShouldNotExistsInTheResponse)
+	ctx.Step(`^the json query "([^"]*)" should exists in the response$`, s.theJsonQueryShouldExistsInTheResponse)
+	ctx.Step(`^the response should not contain a "([^"]*)" set to "([^"]*)"$`, s.theResponseShouldNotContainASetTo)
+}
+
+func (s *ServerTestFeature) theResponseShouldNotContainASetTo(property, value string) error {
+	err := s.TheResponseShouldContainSetTo(property, value)
```

Comment:
What if the property does not exist? This step would pass in this case.

Reply:
I just copy-paste what we have already in the primary api. I will check and re-work if necessary